src/main/vcp/usb_prop.h

```diff
@@ -31,7 +31,7 @@
 
 /* Includes ------------------------------------------------------------------*/
 /* Exported types ------------------------------------------------------------*/
-typedef struct {
+typedef struct __attribute__ ((packed)) {
```

Comment:
Is is really necessary? There may be cleaner way to get correct `sizeof`

Reply:
Yes, this is the correct thing to do if matching a data structure to data on the wire. The alternative is to define a set of constant offsets into a block of data which isn't so friendly.